pom.xml

```diff
@@ -240,16 +240,28 @@
       </plugin>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
-	<artifactId>maven-release-plugin</artifactId>
-	<version>2.5.3</version>
+        <artifactId>maven-release-plugin</artifactId>
+        <version>2.5.3</version>
         <configuration>
           <goals>deploy site site:stage</goals>
         </configuration>
       </plugin>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-site-plugin</artifactId>
-        <version>3.4</version>
+        <version>3.8.2</version>
```

Comment:
We stopped deploying the Maven site long time ago. Either we should document it as part of the release process, or decide to completely drop it (http://sonarsource.github.io/sonar-scanner-maven/)